client/src/components/Workflow/Editor/WorkflowGraph.vue

```diff
@@ -103,6 +105,7 @@ const props = defineProps({
     isInvocation: { type: Boolean, default: false },
     showMinimap: { type: Boolean, default: true },
     showZoomControls: { type: Boolean, default: true },
+    lowerZIndices: { type: Boolean, default: false },
```

Comment:
Not too familiar with this, is this the recommended way to set the z-index ? Naively I'd think you would override a css rule in InvocationGraph.vue ?

Reply:
Actually overriding a css rule is a better idea, I'll do that

Reply:
Done, overriding css alone did it, thank you!